doc/configuration.rst

```diff
@@ -569,34 +629,59 @@ To exclude the functions mentioned above you would use
 instead of any character, if the name is unambiguous you can also write
 ``pyplot.show`` or just ``show``).
 
-Within your sphinx documentation ``.rst`` files, you can easily
-add this reduced version of the Gallery. For example, the rst below adds
-the reduced version of the Gallery for ``numpy.exp``, which includes all
-examples that use the specific function ``numpy.exp``:
 
-.. code-block:: rst
+.. _file_based_minigalleries:
 
-    .. minigallery:: numpy.exp
-        :add-heading:
+Create mini-galleries using file paths
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``add-heading`` option adds a heading for the mini-gallery, which will be a
-default generated message if no string is provided as an argument. The example
-mini-gallery shown above uses the default heading. The level of the heading
-defaults to ``^``, but can be changed using the ``heading-level`` option, which
-accepts a single character (e.g., ``-``). The mini-gallery will only be shown
-if the item (here ``numpy.exp``) is actually used or referred to in an example.
+Sometimes you may want to explicitly create a 
+:class:`mini-gallery <sphinx_gallery.directives.MiniGallery>` using files that 
+do not have functions in common, for example a set of tutorials. The 
+mini-gallery directive therefore also supports passing in:
 
-You can also list multiple items, separated by spaces, which will merge all
-examples into a single mini-gallery, e.g.:
+* pathlike strings to sphinx gallery example files
+* glob-style pathlike strings to sphinx-gallery example files
 
 .. code-block:: rst
 
-    .. minigallery:: numpy.exp numpy.sin
-        :add-heading: Mini-gallery using ``numpy.exp`` or ``numpy.sin``
-        :heading-level: -
+    .. minigallery:: numpy.exp ../examples/plot_0_sin.py ../examples/plot_4*
+
+
+.. minigallery:: numpy.exp ../examples/plot_0_sin.py ../examples/plot_4*
+    :add-heading: Mini-gallery using paths
+    :heading-level: ^
+
+Sort mini-gallery thumbnails from files
+"""""""""""""""""""""""""""""""""""""""
+
+The mini-gallery directive generates a gallery of thumbnails corresponding to 
+the input file strings or object names. The ordering of these thumbnails can be 
+set using the ``minigallery_sort_order`` configuration key. The minigallery 
+directive passes the following information to :py:func:`sorted`:
+
+* sphinx-gallery :ref:`sorting key <sort-warning>` method
```

Comment:
I don't quite follow this. I thought the `sorted` function got the `minigallery_sort_order` function?